templates/mongodb.conf.erb

```diff
@@ -180,3 +180,8 @@ syslog = <%= @syslog %>
 <% if @quiet -%>
 quiet = <%= @quiet %>
 <% end -%>
+<% if @ssl -%>
```

Comment:
This should probably be `and @ssl_key and @ssl_ca` to make sure that they are actually all there. And then you don't actually need `@ssl` at all, though it would probably be better to be explicit rather than implicit.